oak_crypto/src/noise_nk/mod.rs

```diff
@@ -0,0 +1,267 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This was copied from Chromium's third_party/cloud_authenticator, which has
+//! compatible copyright and ownership (Apache 2.0, Google).
+
+#[allow(unused_imports)] // Macros only used in tests.
+#[macro_use]
+
+mod error;
+mod noise;
+mod rustcrypto;
```

Comment:
I think if we want to only use RustCrypto, it could be defaulted and we don't need to explicitly name the corresponding module `rustcrypto` (because it assumes that we may support other crypto libraries)